x-pack/plugin/esql/qa/testFixtures/src/main/resources/string.csv-spec

```diff
@@ -38,6 +38,58 @@ emp_no:integer | l:integer
 10003 | 5
 ;
 
+bitLength
+required_capability: fn_bit_length
+row a = "hello", b = "" | eval y = bit_length(a) + bit_length(b);
+
+a:keyword | b:keyword | y:integer
+hello | | 40
+;
+
+bitLengthWithNonAsciiChars
+required_capability: fn_bit_length
+row a = "¡", b = "❗️" | eval y = bit_length(a) | eval z = bit_length(b);
+
+a:keyword | b:keyword | y:integer | z:integer
+¡ | ❗️ | 16 | 48
+;
+
+foldBitLength
+required_capability: fn_bit_length
+row a = 1 | eval b = bit_length("hello");
+
+a:integer | b:integer
+1 | 40
+;
+
+bitLengthAndSourceQuoting
+required_capability: fn_bit_length
+required_capability: double_quotes_source_enclosing
```

Comment:
Similar as with greatest: As capabilities are barely a "pseudo-versioning system", it's enough with adding the new one you added, as it "implies" the others exist already

Reply:
Thanks for the explanation, makes sense! Adjusted with [Remove FN_GREATEST capability and only use the most recent in bitLeng…](https://github.com/elastic/elasticsearch/pull/115792/commits/666198e7c410df6ea59c9ce8327ada5f0497d375)